model_analyzer/analyzer.py

```diff
@@ -203,6 +170,54 @@ def report(self, mode):
         self._report_manager.create_detailed_reports()
         self._report_manager.export_detailed_reports()
 
+    def _profile_models(self):
```

Comment:
Should change the order of these to match how they are called (reads better)

Reply:
done